### DISCUSSION TITLE
Saving world data: X% Spam

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2559] [Q&A]
To reduce console/log spam, is there a way to suppress the "Saving world data" and related messages? Or at least reduce it to something like "starting saving" and "finished saving"?

115ish lines every few minutes is rather cumbersome...

**Comment by Kojirremer** [comment]
Oh, that would really be nice. Sometimes I'm looking at the console typing commands using the logged lines as reference when suddenly "BAM, saving world, good luck finding the line you were just looking at!"

**Reply by hakusaro** [reply]
I made an issue for this! https://github.com/Pryaxis/TShock/issues/2560

**Comment by pcider** [comment]
I use `grep -a -v -E ".+ [0-9]+%" server.log | less` to filter out the spam, but yeah it's pretty annoying especially when port sniffers trigger world save too. But when piping the server stdout through grep, tshock would reject commands, so I use screen's logging feature to log to `server.log` (but there would be console control characters in the log). This method is pretty hacky though.

**Reply by hakusaro** [reply]
Wow. I’m actually really sorry. I didn’t realize that this spam was that bad to people.

**Reply by pcider** [reply]
no worries, I'm thankful to have TShock since it allows me to play with my mobile friends :)
